Pcap++/header/WinPcapLiveDevice.h

```diff
@@ -55,6 +55,9 @@ namespace pcpp
 		 * setMinAmountOfDataToCopyFromKernelToApplication())
 		 */
 		int getMinAmountOfDataToCopyFromKernelToApplication() const { return m_MinAmountOfDataToCopyFromKernelToApplication; }
+
+	protected:
+		PcapLiveDevice* cloneInternal(pcap_if_t& devInterface) const override;
```

Comment:
nit: I think the signature can change a bit to return`WinPcapLiveDevice*`:
```cpp
WinPcapLiveDevice* cloneInternal(pcap_if_t& devInterface) const override;
```

Reply:
Implemented 62b8b12